### PR TITLE
 microverse portfolio responsive Pull request

### LINKED ISSUE #10: create-header-desktop-version
### Issue: 9-create-header-desktop-version
- I modify the header to make it responsive for the desktop version
- I add 3 nav items for it :  Portfolio, About and Contact
- The hamb icon which is usually seen in Mobile version will now be disabled so users can see the 3 nav items.
- I add the right margins and space to header 
 
### Goal
 - Make the header responsive 
 - Understand how to use **Media-Queries**

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About the project 🌷
- This project is about the creation of the **desktop version** of the mobile version for the project 
- Media-queries >= to a min-width of 768px has been added
- Css Grids and FlexBox has been used to keep the structure of the site for a desktop version
- Figma was a crucial part for this responsive part "We learned how to **parse** Figma design
- A new feature branch is created to work specific in this desktop version
- Stylelint and Webhint are used. They are important to deliver a clean code 
- The usage of Background position and its properties has been used to add the appropiate background to certain images.

### why these tools are important? 📍
- Css Grids, FlexBox, Figma, Webhint, Stylelint, and Media-Queries are essential to deliver a responsive design.

### what was the most complex part? ⭐️
I feel that understand how to parse Figma, and also the usage of CSS Grids were the hardest things to me.
But now I appreciate this learning because it gives me news tools to interact with and deliver responsive designs.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -568,4 +572,472 @@ h3.lang-1 {
...
+
+  p.presentation-paragraph {
+    font-family: Poppins-Thin, sans-serif;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    padding-right: 0;
+    padding-left: 0;
+  }
```
Comment (Qoosim) [inline css/style.css:771]: - [ ] Great work for making the desktop version of your portfolio. You've really put in great works styling your desktop version without affecting your mobile. Kudos to you man :handshake: . There's just little stufff you need to work on in your about me section. According to the Figma design, the paragraph part takes the width of 40% and the language list takes the width of 60% as opposed to your design. Kindly look into the two screenshots and walk your design alongside the Figma design.

### Your Design
your-design

### Figma Design
temp-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @IvonneBenitesRodriguez, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply css/style.css:771]: Dear @Qoosim  👋🏾
Very nice to meet you, thank you for your kind words, and suggestion, I appreciate it.
I modified the width to 40% for the about-myself-first class which contains : the title, the paragraph, link , social icons and buttons. Besides, I gave also a width of 60% to language-list class as you suggested.
I have to mention that all these elements are inside a section named about-myself.
Thanks again,  can you please verify it?
Best regards,

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @IvonneBenitesRodriguez 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Excellent communication with the reviewer

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**